activerecord/lib/active_record/query_logs.rb

```diff
@@ -73,7 +74,9 @@ module QueryLogs
     mattr_accessor :tags, instance_accessor: false, default: [ :application ]
     mattr_accessor :prepend_comment, instance_accessor: false, default: false
     mattr_accessor :cache_query_log_tags, instance_accessor: false, default: false
+    mattr_accessor :tags_format, instance_accessor: false, default: :default
```

Comment:
I'm not sure it makes sense to store both the `format` and the `formatter` on `QueryLogs`, I would think only the `formatter` is necessary

Reply:
Great idea - I was able to remove `tags_format` and kept everything working correctly 😃